workspace-manager-clienttests/src/main/resources/configs/resiliency/DeleteInitialPods.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "DeleteInitialPods",
```

Comment:
Let's call this something that includes what the test is trying to do during the disruption, e.g. CheckStatusWhileDeletePods. My reason for this is that I don't want to give the idea that each DisruptiveScript should get its own test confg -- actually, the disruptive scripts should be used by any test config by just setting the disruptiveScript property.

Reply:
Renamed `DeleteInialPods` in the name field to `CheckStatusWhileDeletePods` as suggested.